### PR TITLE
Coordination meeting minutes, 2021-11-25

### PR DESCRIPTION
**Pull request by graeme-a-stewart**

### CODE REVIEW

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+- Next steps not decided at the moment
+
+### HSF Talks
+
+#### Snowmass Small Experiments Workshop
+
+There was a Snowmass Small Experiments Workshop [last week](https://indico.physics.lbl.gov/event/1756/overview). Liz gave the HSF talk.
+
+- Small experiments does not mean small computing problems
+  - They have with less human resources to tackle the issues
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:42]:
```suggestion
  - They have less human resources to tackle the issues
```

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+- Data management is often at the PB scale, which is well beyond management by hand
+- Great reliance on common software, like ROOT and Geant4
+- Small cosmic experiments are often west coast US based, which is a hard TZ for collaboration with CERN and Europe
+
+One practical follow-up would be in the context of the ESCAPE workflows activity (Caterina is our contact there). We want to follow-up on that next year.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:47]:
```suggestion
One practical follow-up would be in the context of the ESCAPE workflows activity (Caterina is our contact there). We want to follow up on that next year.
```

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+
+There is a follow-up report document to which we can contribute (join the "computingforsmallexps" channel in the Snowmass Slack).
+
+### ASCR Workshop on the Science of Scientific-Software Development and Use
+
+There is a DOE supported [ASCR workshop](https://web.cvent.com/event/1b7d7c3a-e9b4-409d-ae2b-284779cfe72f/summary) 13-15 December on scientific software. There is a call for [position papers](https://web.cvent.com/event/1b7d7c3a-e9b4-409d-ae2b-284779cfe72f/websitePage:5c30ffe5-b577-491f-8d8d-1f745b03e9ec): *We invite community input in the form of **two-page position** papers that identify and discuss key challenges and opportunities in the science of the scientific-software development process and the study of the use of scientific software.*
+
+It was felt to useful to submit a paper to this, so Graeme, Liz and Eduardo wrote/contributed to [what we submitted](https://docs.google.com/document/d/1f3wcOgE8tqh-Q88A3wzQrZaWhLazPl196g1lLkv0XZo/edit?usp=sharing).
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:55]:
```suggestion
It was felt useful to submit a paper to this, so Graeme, Liz and Eduardo wrote/contributed to [what we submitted](https://docs.google.com/document/d/1f3wcOgE8tqh-Q88A3wzQrZaWhLazPl196g1lLkv0XZo/edit?usp=sharing).
```

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+
+- We seem to be more of an "interesting problem" than a solution in this space, but no bad idea to throw our hat into the ring.*
+
+## Working Group Updates
+
+### Mandates for 2022
+
+We have started to think about WG convenerships for 2022. For WG convenors, please let Benedikt know your plans.
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:63]:
```suggestion
We have started to think about WG convenerships for 2022. For WG conveners, please let Benedikt know your plans.
```

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 5]
```diff
@@ -0,0 +1,168 @@
...
+- Presentation on December 13th on FPGAs for simulation:
+  - <https://indico.cern.ch/event/1096432/>
+- Now thinking about/planning meetings for 2022
+- Talk on simulation at Snowmass small experiment workshop
+  - <https://indico.physics.lbl.gov/event/1756/contributions/6251/>
+  - Noted that all small experiments rely heavily on Geant4 and need it to be sustained by the FAs
+
+### Reconstruction and Software Triggers
+
+- One outcome of 2021 survey for Reconstruction was the need/request for R&D detector dataset:
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:96]:
```suggestion
- One outcome of 2021 survey for Reconstruction was the need/request for R&D detector datasets:
```

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 6]
```diff
@@ -0,0 +1,168 @@
...
+  - Open Data Detector - Tracking System (follow-up of TrackML detector) to be ACTS-free (MR + ACAT poster)
+  - Calorimeter system will be added as part of a bigger initiative of CERN EP-R&D allows full Geant4 simulation of realistic tracking (including timing) + calorimeter
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:98]: Not sure how to rephrase the sentence without changing the present meaning, which is a bit fuzzy.

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 7]
```diff
@@ -0,0 +1,168 @@
...
+- Also a [HSF PHOENIX](https://acts-project.github.io/odd-phoenix/) version available.
+
+### PyHEP
+
+- The last topical "Python Module of the Month" meeting of the year will take place on December 8th and will be devoted to PyTorch. Agenda at <https://indico.cern.ch/event/1098618/>.
+  - Training frameworks were quite disucssed at the LPCC Fast Sim workshop this week
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:104]:
```suggestion
  - Training frameworks were quite discussed at the LPCC Fast Sim workshop this week.
```

File: organization/_posts/2021/2021-11-25-coordination.md [hunk 8]
```diff
@@ -0,0 +1,168 @@
...
+
+### Software Tools and Packaging
+
+- Nothing new to report.
+
+### Software Training
+
+- Next Sotware Carpentries workshop in preparation. Link to [Indico page](https://indico.cern.ch/event/1097111/)
```
Comment (eduardo-rodrigues) [inline organization/_posts/2021/2021-11-25-coordination.md:112]:
```suggestion
- Next Software Carpentries workshop in preparation. Link to [Indico page](https://indico.cern.ch/event/1097111/)
```

**Review by eduardo-rodrigues — APPROVED** [review]
Some suggestion and trivial fixes. Advance approval going out :-).

File: organization/_posts/2021/2021-11-25-coordination.md (hunk 6 above)
Comment (graeme-a-stewart) [reply organization/_posts/2021/2021-11-25-coordination.md]: I had a go at making it clearer

**Comment by graeme-a-stewart** [comment]
> Some suggestion and trivial fixes. Advance approval going out :-).

Thanks for the careful reading @eduardo-rodrigues !

**Comment by valassi** [comment]
Hi Graeme sorry I missed the review notification. No comments from me... thanks for the minutes

**Pull request merged**